### PR TITLE
Add forms 📝

### PR DESCRIPTION
**Pull request by hydaline-code**
Created a method called current_user in  ApplicationController 
Created some  forms to perform the following functions:
✔Creates a Post for  current_user 
✔Creates a comment for  current_user 
kindly ♻ 🙏

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_user ||= User.first
+  end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/application_controller.rb:4]: - [ ] Kudos for adding this requirement

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        format.html { render :new }
+      end
+    end
+  end
+
+  private
+
+  def comment_params
+    puts params
+    params.require(:comment).permit(:text)
```
Comment (VICTOR-CHUKWU) [inline app/controllers/comments_controller.rb]: - [ ] Impressive lines of code that meet the requirements

File: app/views/posts/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<%= form_with( url: user_posts_path(@user), method: :post) do |form| %>
+  <%= form.label :title %>
+  <%= form.text_field :title %>
+
+  <%= form.label :text %>
+  <%= form.text_area :text %>
+
+  <%= form.submit 'Create Post' %>
+<% end %>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/new.html.erb]: ##Required

- [ ]  Kindly make this form to be displayed as in the wireframes. It is required to accept input for creating a post

File: app/views/comments/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<%= form_with( url: post_comments_path(@post), method: :post) do |form| %>
+  <%= form.label :text %>
+  <%= form.text_area :text %>
+
+  <%= form.submit 'Add Comment' %>
+<% end %>
```
Comment (VICTOR-CHUKWU) [inline app/views/comments/new.html.erb]: ## Required

- [ ] Kindly make this form to be displayed as in the wireframes. It is required to accept input for creating comments

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @hydaline-code  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed
- You have implemented the greater part of this milestone

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — COMMENTED** [review]
Hi @hydaline-code,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* This is a new week and the project was not approved last week. Kindly submit a new pull request without previous reviewer comments.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**